.gitignore

```diff
@@ -18,4 +18,6 @@ atlassian-ide-plugin.xml
 /target/
 src/main/javascript/node
 src/main/javascript/node_modules
-src/main/resources/generated
\ No newline at end of file
+src/main/resources/generated
+
+ci/node_modules
```

Comment:
Instead of having multiple `node_modules` added to gitignore we could add `**/node_modules` to ignore all of them at once.